docs/articles/expensify-classic/get-paid-back/expenses/Merge-Expenses.md

```diff
@@ -1,5 +1,62 @@
 ---
-title: Merge Expenses
-description: Merge Expenses
+title: Merge-Expenses.md
```

Comment:
Did someone say we should do this? I'm actually not familiar with this but it feels weird to have the filename not a readable title....

Reply:
Hmm it's possible I misunderstood something but here's the SO I have been referencing: https://stackoverflowteams.com/c/expensify/questions/15135

Reply:
Yeah so I **think** they're just talking about the name of the file itself (which is correctly `Merge-Expenses.md` - I think the `title` here should be user-friendly. Maybe make the change and if it's wrong you can blame me :)

Reply:
Will do!